pkg/storageincentives/agent.go

```diff
@@ -129,8 +129,6 @@ func (a *Agent) start(blockTime time.Duration, blocksPerRound, blocksPerPhase ui
 			a.logger.Error(err, "phase failed", "phase", phase, "round", round)
 		} else if isPhasePlayed {
 			a.logger.Info("phase played", "phase", phase, "round", round)
-		} else {
```

Comment:
Was this informative log removed on purpose?

Reply:
yeah, it's unnecessary to log that the node is not playing the round in every phase